### PR TITLE
ICDS Dashboard: more topojson cleanup

### PR DESCRIPTION
**Pull request by czue**
Additional follow up and cleanup maps based on https://github.com/dimagi/commcare-hq/pull/26242

This does a number of things:

1. Automates the process of downsizing topojsons via a management command.
2. Ports mobile dashboard and feature flag maps to use topojson files instead of javascript files that start with `var TOPOJSON = [actual data]`
3. Moves all topojson files back into `custom/icds_reports/static/js/topojsons`
4. Adds optimized topojsons for district-level maps (~10 times smaller file)
5. Removes legacy topojson files that were no longer used anywhere

One neat thing about point 2 is that github now renders topojson files which is super cool.

https://github.com/dimagi/commcare-hq/blob/b1dc730cbdebb9b52932df2b65ab4843b8dbff35/custom/icds_reports/static/js/topojsons/states_v3_small.topojson

Review by commit. And for intimidated reviewers, the actual code changes here are very minimal (just `indie-map` and `topojson.service` really).

### CODE REVIEW

**Comment by czue** [comment]
ooh this is gonna be good for my stats... 
image

File: custom/icds_reports/static/js/angular-services/topojson.service.js [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
+                return Promise.resolve(CACHE["blocks"][district]);
+            } else {
+                return $http.get(url('topojson'), {
+                    params: {district: district},
+                }).then(
+                    function (response) {
+                        CACHE['blocks'][district] = response.data;
+                        return response.data;
+                    },
+                );
```
[the remark this answers is not in the document]
Comment (snopoke) [reply custom/icds_reports/static/js/angular-services/topojson.service.js]: is this correct?

**Review by snopoke — APPROVED** [review]
Looks good to me

File: custom/icds_reports/static/js/angular-services/topojson.service.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (czue) [reply custom/icds_reports/static/js/angular-services/topojson.service.js]: It was, I deleted the comma in https://github.com/dimagi/commcare-hq/pull/26252/commits/b1dc730cbdebb9b52932df2b65ab4843b8dbff35

**Pull request merged**